pkg/apis/feature/features_test.go

```diff
@@ -56,3 +57,14 @@ func TestGetFlags(t *testing.T) {
 	require.True(t, flags.IsAllowed("my-allowed-flag"))
 	require.False(t, flags.IsAllowed("non-disabled-flag"))
 }
+
+func TestShouldNotOverrideDefaults(t *testing.T) {
+
+	f, err := NewFlagsConfigFromMap(map[string]string{})
+	require.Nil(t, err)
+	require.NotNil(t, f)
+
+	if !f.IsDisabled(KReferenceGroup) && !f.IsEnabled(KReferenceGroup) {
```

Comment:
@pierDipi can we also add a check here for one of the "enabled" by default flags?